Scientist4JCore/src/main/java/com/github/rawls238/scientist4j/Experiment.java

```diff
@@ -25,29 +28,29 @@
     private final Counter totalCount;
 
     public Experiment() {
-      this("Experiment");
+        this("Experiment");
     }
 
     public Experiment(String name) {
         this(name, false);
     }
 
     public Experiment(String name, Map<String, Object> context) {
-      this(name, context, false);
+        this(name, context, false);
     }
 
     public Experiment(String name, boolean raiseOnMismatch) {
-      this(name, new HashMap(), raiseOnMismatch);
+        this(name, new HashMap<>(), raiseOnMismatch);
     }
 
     public Experiment(String name, Map<String, Object> context, boolean raiseOnMismatch) {
-      this.name = name;
-      this.context = context;
-      this.raiseOnMismatch = raiseOnMismatch;
-      controlTimer = metrics.timer(MetricRegistry.name(NAMESPACE_PREFIX, this.name, "control"));
-      candidateTimer = metrics.timer(MetricRegistry.name(NAMESPACE_PREFIX, this.name, "candidate"));
-      mismatchCount = metrics.counter(MetricRegistry.name(NAMESPACE_PREFIX, this.name, "mismatch"));
-      totalCount = metrics.counter(MetricRegistry.name(NAMESPACE_PREFIX, this.name, "total"));
+        this.name = name;
```

Comment:
I tired all combinations of tab/indent of 4/2 and can't get it to do what you are asking.

<img width="930" alt="screen shot 2016-05-02 at 10 21 22 am" src="https://cloud.githubusercontent.com/assets/691357/14958296/b9fe0906-104f-11e6-9761-de78e4bd398a.png">